README.md

```diff
@@ -146,29 +129,29 @@ In the node level, this plugin collects metrics listed the next catalog.
 |script| Computing the grades stats based on a script|
 
 ### Examples
-Example running snap-plugin-collector-elasticsearch, passthru processor, and writing data to a file.
+Example running snap-plugin-collector-elasticsearch and writing data to a file.
 
 ![Dockerized example](http://media.giphy.com/media/3osxY87TeMy7jGrbDW/giphy.gif)
```

Comment:
About the gif:
- It's still using $SNAP_PATH.
- It's using `snapd/snapctl`.

Do we need this gif?
If yes, can we update it to the new example?